lua/telescope/utils.lua

```diff
@@ -249,95 +249,6 @@ function utils.get_os_command_output(cmd, cwd)
   return stdout, ret, stderr
 end
 
-utils.strdisplaywidth = (function()
```

Comment:
Can we not just remove these functions but do something like this
```lua
utils.x = function()
  error("x deprecated. please use plenary.strings.x") 
end
```

Reply:
Added on fd1a736